R/utilities-writing-report.R

```diff
@@ -445,3 +447,23 @@ getSimulationDescriptor <- function(workflow) {
   validateIsOfType(workflow, "Workflow")
   return(workflow$getSimulationDescriptor())
 }
+
+#' @title adjustTitlePage
+#' @description Adust Qualification Version Information to be displayed on title page 
+#' @param fileName name of .md file to update
+#' @param qualificationVersionInfo A `QualificationVersionInfo`object defining Qualification Version Information to be displayed on title page
+#' @export
+adjustTitlePage <- function(fileName, qualificationVersionInfo = NULL) {
+  validateIsOfType(qualificationVersionInfo, "QualificationVersionInfo", nullAllowed = TRUE)
+  # Does not adust title page if no QualificationVersionInfo
+  if(isEmpty(qualificationVersionInfo)){
```

Comment:
isEmpty or isNull?
Or is it the same here?

Reply:
They will work the same way here